crustcrawler_arm/ArmPose arm_pose
float32[6] raw_positions
float32[6] degree_positions
